zabbix_metrics_watcher.py

```diff
@@ -187,8 +187,12 @@ def filter_completed_builds(completed_builds):
 
 
 def run(zabbix_host, osbs_master, config, instance):
+    STUCK_IN_NEW_THRESHOLD_SECS = 5
+
     running_builds = set()
+    builds_in_new = set()
```

Comment:
I think this should be a dict of build_name -> time_first_seen.

Then we can do:
```
now = datetime.datetime.now()
if build_name not in builds_in_new:
    builds_in_new[build_name] = now

spent_in_new = [(now - start).total_seconds() for start in builds_in_new.values()]
if spent_in_new and max(spent_in_new) > STUCK_IN_NEW_THRESHOLD_SECS:
    ...
```